polaris/hub/polarisfs.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+
+import fsspec
+import httpx
+
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+from polaris.utils.errors import PolarisHubError
+
+
+class PolarisfsSettings(BaseSettings):
+    url: str = "http://localhost:3000/"
+
+    model_config = SettingsConfigDict(env_prefix="POLARIS_")
```

Comment:
I don't think we need this? Hadrien needed this as he was using a newly created `httpx` client in his request. We can rely on the Polaris client, which already has the relevant settings.